src/SDKs/DevTestLabs/DevTestLabs.Tests/Tests/BasicTests.cs

```diff
@@ -20,7 +20,7 @@ public void ListLabsTest()
             {
                 var client = GetDevTestLabsClient(context);
 
-                var labs = client.Lab.ListBySubscription().ToList();
+                var labs = client.Labs.ListBySubscription().ToList();
```

Comment:
@ryanvog the tests I am referring to are for the models and it's associated operations you have exposed in the DevTestLab SDK.
I was referring to testing the SDK itself. I don't see any test that will test:
- VNetOperations
- ScheduleOperation
- Setting Policy Operation
- Lab Operation
Plus any applicable tests that will test your exposed model.